x-pack/plugins/security/public/views/management/edit_role/components/privileges/kibana/__snapshots__/privilege_space_form.test.tsx.snap

```diff
@@ -1,101 +1,119 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`<PrivilegeSpaceForm> renders without crashing 1`] = `
-<EuiFlexGroup
-  alignItems="stretch"
-  component="div"
-  direction="row"
-  gutterSize="l"
-  justifyContent="flexStart"
-  responsive={false}
-  wrap={false}
->
-  <EuiFlexItem
-    component="div"
-    grow={true}
-  >
-    <EuiFormRow
-      describedByIds={Array []}
-      fullWidth={false}
-      hasEmptyLabelSpace={false}
-      isInvalid={false}
-      label={
-        <FormattedMessage
-          defaultMessage="Spaces"
-          id="xpack.security.management.editRoles.privilegeSpaceForm.spacesFormRowLabel"
-          values={Object {}}
-        />
-      }
-    >
-      <SpaceSelector
-        onChange={[Function]}
-        selectedSpaceIds={Array []}
-        spaces={
-          Array [
-            Object {
-              "_reserved": true,
-              "description": "",
-              "id": "default",
-              "name": "Default Space",
-            },
-            Object {
-              "description": "",
-              "id": "marketing",
-              "name": "Marketing",
-            },
-          ]
-        }
-      />
-    </EuiFormRow>
-  </EuiFlexItem>
-  <EuiFlexItem
-    component="div"
-    grow={true}
-  >
-    <EuiFormRow
-      describedByIds={Array []}
-      fullWidth={false}
-      hasEmptyLabelSpace={false}
-      isInvalid={false}
-      label={
-        <FormattedMessage
-          defaultMessage="Privilege"
-          id="xpack.security.management.editRoles.privilegeSpaceForm.privilegeFormRowLabel"
-          values={Object {}}
-        />
-      }
-    >
-      <PrivilegeSelector
-        availablePrivileges={
-          Array [
-            "all",
-            "read",
-          ]
-        }
-        data-test-subj="space-form-priv-selector"
-        onChange={[Function]}
-        value="none"
-      />
-    </EuiFormRow>
-  </EuiFlexItem>
-  <EuiFlexItem
-    component="div"
-    grow={false}
-  >
-    <EuiFormRow
-      describedByIds={Array []}
-      fullWidth={false}
-      hasEmptyLabelSpace={true}
-    >
-      <EuiButtonIcon
-        aria-label="Delete space privilege"
-        color="danger"
-        iconSize="m"
-        iconType="trash"
-        onClick={[MockFunction]}
-        type="button"
-      />
-    </EuiFormRow>
-  </EuiFlexItem>
-</EuiFlexGroup>
+<PrivilegeSpaceFormUI
```

Comment:
Similar to my comment on #26082, this test is no longer testing the actual form component, only the Intl wrapper. Needs to be testing the actual form component.